arch/freertos/ua_architecture.h

```diff
@@ -0,0 +1,96 @@
+/* This work is licensed under a Creative Commons CCZero 1.0 Universal License.
+ * See http://creativecommons.org/publicdomain/zero/1.0/ for more information.
+ *
+ *    Copyright 2016-2017 (c) Julius Pfrommer, Fraunhofer IOSB
+ *    Copyright 2017 (c) Stefan Profanter, fortiss GmbH
```

Comment:
We are in 2018 right now :-)

Reply:
Ah, thanks for the hint, my head was still in 2017, thanks to Copy & Paste 😄 
We also need to add @cabralfortiss to the list. But there is still some WIP for this PR.